eng/Versions.props

```diff
@@ -1,17 +1,19 @@
 ﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project>
   <PropertyGroup>
-    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
-
+    <RepositoryUrl>https://github.com/dotnet/diagnostics</RepositoryUrl>
```

Comment:
Does `RepositoryUrl` belong on the `Versions.props`?

Reply:
Yes, that's the usual thing in other repos. Not really sure why.